src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -569,6 +570,47 @@ trait WorkflowComponent {
       deleteWorkflowFailureAttributes(workflowFailureId) andThen
         findInputResolutionsByFailureId(workflowFailureId).delete
     }
+
+
+    object WorkflowStatisticsQueries extends RawSqlQuery {
+      val driver: JdbcDriver = WorkflowComponent.this.driver
+
+      def countWorkflowsPerUserQuery(startDate: String, endDate: String) = {
+        sql"""select min(count), max(count), avg(count), stddev(count)
+                from (
+                  select count(1) as count from WORKFLOW w
+                    join SUBMISSION s on s.ID=w.SUBMISSION_ID
+                    where s.DATE_SUBMITTED between $startDate and $endDate
+                    group by s.SUBMITTER
+                ) as counts""".as[SummaryStatistics].head
+      }
+
+      def countWorkflowsPerSubmission(startDate: String, endDate: String) = {
+        sql"""select min(count), max(count), avg(count), stddev(count)
+                from (
+                  select count(1) as count from WORKFLOW w
+                    join SUBMISSION s on s.ID=w.SUBMISSION_ID
+                    where s.DATE_SUBMITTED between $startDate and $endDate
+                    group by w.SUBMISSION_ID
+                ) as counts""".as[SummaryStatistics].head
+      }
+
+      def workflowRunTimeQuery(startDate: String, endDate: String) = {
+        sql"""select min(seconds), max(seconds), avg(seconds), stddev(seconds)
+                from (
+                  select TIMESTAMPDIFF(SECOND, MIN(a.timestamp), MAX(a.timestamp)) as seconds
+                    from WORKFLOW w join AUDIT_WORKFLOW_STATUS a on w.ID=a.workflow_id
+                    where w.STATUS in ("Succeeded") and a.STATUS in ("Queued","Succeeded")
```

Comment:
Is there a way to guarantee that the audit table returns both a Queued and Succeeded row?  If there's only one, it will add spurious zeroes to the stats.  Maybe that's how to do it: exclude diff=zero